homeassistant/components/update/__init__.py

```diff
@@ -308,6 +308,14 @@ def state_attributes(self) -> dict[str, Any] | None:
         else:
             in_progress = self.__in_progress
 
+        # Clear skipped version in case it matches the current version or
+        # the latest version diverged.
+        if (
+            self.__skipped_version == self.current_version
+            or self.__skipped_version != self.latest_version
+        ):
+            self.__skipped_version = None
```

Comment:
This makes me wonder @ludeeus, should the `SKIPPED_VERSION` just be `SKIPPED` and a boolean in the attributes below? As in, is there a use case to have the specific version that is skipped exposed?

Reply:
Since it always would be the current latest version or None, no not really 👍

Reply:
Oh... there is.
If the latest version is updated (the value) externally, we want to not have that new one marked as skipped

Reply:
Oh right 👍